packages/react-components/jest.config.js

```diff
@@ -17,5 +17,4 @@ module.exports = {
   },
   coverageDirectory: './coverage',
   setupFilesAfterEnv: ['./config/tests.js'],
-  snapshotSerializers: ['@fluentui/jest-serializer-make-styles'],
```

Comment:
can we leave this here as well for sake of consistency ? If anyone adds any kind of makeStyles suddenly test will start failing. I'm not seeing any downsides to have this in place. WDYT?

Reply:
I restored this, but as Griffel packages. This package is the last that has this (`@fluentui/jest-serializer-make-styles`) dependency.